packages/docs/src/page-configs/ui-elements/virtual-scroller/examples/DifferentContent.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <va-virtual-scroller
+    :items="hugeArray"
+    style="height: 400px;"
```

Comment:
That doesn't look nice, if hight/width is mandatory - we should make it to a prop for convenience.